native/src/routes/PDFViewModal.tsx

```diff
@@ -38,6 +38,7 @@ const PDFViewModal = ({ route, navigation: _navigation }: PropsType): ReactEleme
         source={{
           uri: url,
         }}
+        trustAllCerts={false}
```

Comment:
Could you please explain this? Why does this help?

Reply:
Some bug in react-native-pdf or react-native-fetch-blob. Should be default be false, but does not work correctly. See here:
https://github.com/wonday/react-native-pdf/issues/460

Reply:
Thanks! According to the docs the default is true.